lib/authentication/model/user.dart

```diff
@@ -6,25 +6,15 @@ class User {
   String firstName;
   String lastName;
 
-  String degree;
-  String domain;
-  String year;
-  String series;
-  String group;
-  String subgroup;
+  List<String> classes;
```

Comment:
The name is bad but I don't have a better idea, so let's at least add some doc to it.
```suggestion
    /// Info about the user's assigned group (including degree, year of study, series etc)
    List<String> classes;
```